### ISSUE TITLE
2.6.1 LRScheduler set_state_dict 出现多余warning

### ISSUE THREAD

**Issue by johnyanccer** [PaddlePaddle/Paddle#65194]
### bug描述 Describe the Bug

2.6.1（及以前版本）均出现此warning，都属于不必要的信息

设置两个LRScheduler 嵌套的情形，如
schedulerX = paddle.optimizer.lr.PolynomialDecay(learning_rate=0.5, decay_steps=20, verbose=True)
scheduler = paddle.optimizer.lr.LinearWarmup(
            ...         learning_rate=schedulerX, warmup_steps=20, start_lr=0, end_lr=0.5, verbose=True)
此时对scheduler进行set_state_dict，就会出现warning
 warnings.warn(
                "There are some unused values in state_dict. Maybe the optimizer have different 'LearningRateDecay' when invoking state_dict and set_dict"）
但其实并没有出现warn提示的情形（state_dict中所有values都用到了）

原因在于17个scheduler策略的set_state_dict都直接调用了基类LRScheduler的method :
def set_state_dict(self, state_dict):
    super().set_state_dict(state_dict)，

基类的state_dict只有2个keys：['last_epoch', 'last_lr']，子策略的state_dict将会多出一个key: "LinearWarmup_LR"
if isinstance(self.learning_rate, LRScheduler):
            state_dict["LinearWarmup_LR"] = self.learning_rate.state_dict() 
所以直接调用基类的method set_state_dict就会报warning，提示有的key和value没生效，其实都生效了

建议改一下逻辑，不再报告无效的warning

**Comment by jiahy0825** [comment]
感谢反馈，您对问题已经定位的很清晰了，能否提个修复 PR 出来呢？

**Comment by johnyanccer** [comment]
作为调参侠，非常感谢paddle也很想对paddle框架有贡献，无奈git使用太生疏（完全用不到），github网站访问也比较困难，没法提供修复PR了 



> 感谢反馈，您对问题已经定位的很清晰了，能否提个修复 PR 出来呢？
